### ISSUE TITLE
Improve logging

### ISSUE THREAD

**Issue by nsheff** [pepkit/pephub#64]
Please make pepdbagent and pephub use consistent loggers. Probably logmuse will help with this. 

- [x] use consistent logging format
- [x] no `print` messages, always use logger
- [x] use `logmuse` if it makes sense
- [x] improve message for `Invalid token type` message
- [x] track incoming IP addresses in logs if possible
- [x] Fix issue with ~BEDbase logs~ peppy

I see these messages littered all over the logs:

```
Not all environment variables were populated in derived attribute source: $BEDBASE_DATA_PATH_HOST/bed_files/{file_name}
Not all environment variables were populated in derived attribute source: $BEDBASE_DATA_PATH_HOST/outputs/bedstat_output/bedstat_pipeline_logs/submission/{sample_name}_sample.yaml
Not all environment variables were populated in derived attribute source: $BEDBASE_DATA_PATH_HOST/openSignalMatrix_{genome}_percentile99_01_quantNormalized_round4d.txt.gz
```

So basically the point here is: run this service and look at the logs and make the logs consistent and useful and understandable, and correct any errors.

**Comment by khoroshevskyi** [comment]
1) I can make the same style of logging everywhere, but unfortunately, I would have to rewrite all our packages that are used in pephub. None of them have option to pass logger as argument, or set logging level.

e.g. in Samples class. It's defined before class was created:
https://github.com/pepkit/peppy/blob/ca8aa90c161dee0d515cf54c9451081734323d7f/peppy/sample.py#L25
And this is issue with peppy (the last task)

Do you have any suggestion what to do in this case?

2) I checked packages, and all of them are using logger (peppy), or logmuse to log information. I haven't find any print functions.

3) I will work on `Invalid token type` and incoming IP addresses

**Comment by khoroshevskyi** [comment]
To track ip-adress we can simply run this command: request.client.host
More information is here: https://stackoverflow.com/questions/60098005/fastapi-starlette-get-client-real-ip

**Comment by nleroy917** [comment]
@Khoroshevskyi it seems like you might have solved this, is that correct?

**Comment by khoroshevskyi** [comment]
Yes, I solved it. If there is anything else you want to add to logging let me know

**Comment by nleroy917** [comment]
Im going to keep this open since we are discussing the logging capabilities of `uvicorn`

**Comment by nleroy917** [comment]
This was fixed in the latest release
